middlewares/accesslog/logger.go

```diff
@@ -214,6 +214,13 @@ func (l *LogHandler) logTheRoundTrip(logDataTable *LogData, crr *captureRequestR
 	} else {
 		core[Overhead] = total
 	}
+	if crr != nil && crw != nil {
+		core[ResponseDuration] = crw.processingStart.Sub(crr.processingEnd)
```

Comment:
Won't this lead to a negative number in `ResponseDuration`? Start should be smaller than end.

Reply:
It is a bit misleading (perhaps naming should be better in this case). My understanding of the flow is as following:
`crw` is `ResponseWritter` and `processingStart` indicates when Traefik started generating response. `crr` is `RequestReader` where `processingEnd` indicates when Traefik finished processing request body. The only way this will be negative when Generating response will be faster than finishing of request processing... timeouts perhaps?.